docs/UserGuide.md

```diff
@@ -2,133 +2,122 @@
-
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
-
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
+The user can also view more information and examples of using a particular command.
+Format: `help <Command Name>`
+
+Example:
+* `help add`
+Displays more ways(eg: shortcuts) to use the add command 
+
+#### Adding a Task: `add`
+Adds a task to the task manager<br>
+Format : `add <task name>`<br>
+ 	 `add <task name> by <date/day>` <br>
+	 `add <task name> by <date/day> at <time>`<br>
+
+Date Format: dd/mm/yyyy
+Time Format: 12-hour clock ( Eg: 10am, 5pm, etc)
+
+> If the date and time are not mentioned, the task is passed as a “Floating Task” and would be displayed separately.
+> For the date/day, the user can type either the date, which would add the task to that specific day or type the day, which would add the task to the nearest day of the week which matches ‘day’
+
+Examples:
+* `add Read Harry Potter`
+* `add CS2103T Tutorial by 01/10/2016`
+* `add CS2101 reflection by Friday` 
+	(if you type this on a Wednesday, it marks the slot for the coming Friday, that is, day after tomorrow)
+* `add EE2021 assignment by next Thursday`
+	(if you type this on a Wednesday, it marks the slot not for the next day, but for the Thursday after that, that is, 8 days later)
+* `add EE2020 project by 30/10/2016 at 6pm`
+
+#### Priority tasks: `priority`
+Allows the user to set the priority of a task when creating it.<br>
+Format : `priority <priority number>`
+
+> Priority 1 or 2 or 3 is based on how important the task is, with 3 being the highest priority and 1 being the lowest. It should always follow the add command.
+
+Example:
+* `add CS2101 by 01/01/2016 priority 3`
+
+#### Undo a Task: `undo`
+Undo the most recent task entered in the command line<br>
+Format: `undo`
+
+#### Deleting a Task: `delete`
+Deletes a particular task in the task manager<br>
+Format: `delete <task index>`
+
+Example:
+* `delete s1`
+
+#### View: `donelist`
```

Comment:
This feature can be merged with the feature `home`, by using a consolidated command, say `show <content>`. Then you can add more options, like `show <date>` to display tasks on a specific day.